website/i18n/en/docusaurus-plugin-content-docs/current/concepts/low-coupling.md

```diff
@@ -103,25 +103,25 @@ The page data model will be organized as a **composition of features and entitie
 > However, the implementation in the form of a factory is optional - the feature may depend on the underlying layers and directly
 
 ```ts title=pages/main/model.ts
-import { userStore } from "entitites/user"
-import { conversationStore } from "entities/conversation"
-import { contactStore } from "entities/contact"
+import { userModel } from "entitites/user"
```

Comment:
Maybe we should discuss the PascalCase format for architectural modules' names? Like segments, slices, etc?

Reply:
Out-of-scope 🤷‍♂️ 

> Go to slack?